views/food.erb

```diff
@@ -0,0 +1,78 @@
+<html>
+<body>
+  <h1> Seattle tastes good. Best Food, IMO. </h1>
+<img src  = "selam.png" width="300" height="200" />
+<img src  = "maneki.png" width="300" height="200" />
+
+<table border="1">
+  <thead>
+
+    <tr>
+      <th>
+        Best...
+      </th>
+
+      <th>
+      The Spot
+      </th>
+    </tr>
+  </thead>
+
+  <tr>
+    <td style="text-align: center">
+      Best Burrito
+    </td>
+
+    <td>
+      <a href="http://www.yelp.com/biz/el-kiosko-renton?osq=Mexican+Food"> El Kiosko</a>
+    </td>
+  </tr>
+
+  </tr>
+
+  <tr>
+    <td style="text-align: center">
+    Best Dim Sum
+    </td>
+    <td>
+        <a href="http://www.yelp.com/biz/jade-garden-seattle"> Jade Garden</a>
+    </td>
+
+  </tr>
+
+  <tr>
+    <td style="text-align: center">
+    Best Sushi
+    </td>
+    <td>
+        <a href="http://www.yelp.com/biz/maneki-seattle"> Maneki </a>
+    </td>
+
+  </tr>
+  <tr>
+    <td style="text-align: center">
+    Best Ethiopian
+    </td>
+    <td>
+        <a href="http://www.yelp.com/biz/cafe-selam-seattle?osq=Ethiopian+Food"> Cafe Selam </a>
+    </td>
+
+  </tr>
+
+
+</table>
+
+<footer>
+
+  <p>
+    Get your Seattle on:
+    <a href = "/loveseattle.html">
+    Home</a>
+    <a href ="/seattleparks.html">
+    Parks</a>
+
```

Comment:
Also watch out for extra unnecessary whitespace - I'd remove any lines where you have two lines of whitespace